exercises/all-your-base/all_your_base_test.rb

```diff
@@ -1,9 +1,10 @@
-gem 'minitest', '>= 5.0.0'
```

Comment:
Just wanted to double check that it's okay to remove this based on your comment on #509? Thanks!